### PR TITLE
Moving Customize Magento Admin Design to DevDocs

### LINKED ISSUE #8297: Move and update the content from Magento Support KB to DevDocs
In Magento Support Knowledge Base there's an article, which would better belong in Magento Devdocs: 
https://support.magento.com/hc/en-us/articles/360005032794 
So it would be great to "move" it to Devdocs, and updating if required.
High-level steps to complete this issue:

1. Take a look at the article and verify it's content. 
2. Create a new article in the relevant section of the Devdocs repo and copy the updated content from Support KB article. 
 
(this request is approved by Support KB team)

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request

This pull request (PR) fixes #8297 

## Affected DevDocs pages
New Page - https://devdocs.magento.com/guides/v2.3/howdoi/admin/customize_admin.html
New Page - https://devdocs.magento.com/guides/v2.4/howdoi/admin/customize_admin.html

whatsnew
Added a new topic describing how to [customize the Magento Admin design](https://devdocs.magento.com/guides/v2.4/howdoi/admin/customize_admin.html).

### CODE REVIEW

**Comment by BarnyShergold** [comment]
I've moved the article and corrected a lot of the language but it needs fresh eyes! I'm word blind on this one :)

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
+group: how-do-i
+subgroup: admin-customize-page
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:3]:
```suggestion
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+title: Customize Magento Admin Design
+menu_order: 3
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:5]:
```suggestion
```

File: src/_data/toc/how-do-i.yml [hunk 3]
```diff
@@ -55,3 +55,6 @@ pages:
 
       - label: Customize the list of shipping methods
         url: /howdoi/checkout/checkout_shipping_methods.html
+
+  - label: Customize Magento Admin Design
```
Comment (jeff-matthews) [inline src/_data/toc/how-do-i.yml:59]:
```suggestion
  - label: Customize Magento Admin design
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
+---
+group: how-do-i
+subgroup: admin-customize-page
+title: Customize Magento Admin Design
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:4]:
```suggestion
title: Customize Magento Admin design
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+Learn how to customize (add, delete, change) the configuration options available in the Magento Admin. These options define the various aspects of storefront design.
+
+The location of your customizations appears here:
+
+-  Magento Commerce and Open Source 2.1.0 and later: Options set under **CONTENT > Design > Configuration**
+
+## Environment and technologies
+
+-  Magento Commerce
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:16]:
```suggestion
-  {{ site.data.var.ee }}
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+-  Magento Commerce (Cloud)
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:17]:
```suggestion
-  {{ site.data.var.ece }}
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+-  Magento Open Source
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:18]:
```suggestion
-  {{ site.data.var.ce }}
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+
+-  Magento installed
+-  Credentials or access to Magento Admin
+
+## Steps
+
+ {:.bs-callout-info}
+The following walk-through uses version 2.1.0 and later.
+
+In Magento out of the box, when you navigate to **CONTENT > Design > Configuration** in Admin, the first page that opens displays a grid with the available configuration scopes and assigned themes. It looks like following:
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:30]:
```suggestion
On a default Magento installation, when you navigate to **CONTENT > Design > Configuration** in Admin, the first page that opens displays a grid with the available configuration scopes and assigned themes. It looks like following:
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 9]
```diff
@@ -0,0 +1,233 @@
...
+
+![Design Configuration]({{ site.baseurl }}/common/images/design_conf1.png)
+
+When you click **Edit** in any of the scope records, the page with available design options is displayed. For example, the default set of design options for the store view level is the following:
+
+![Default Store View]({{ site.baseurl }}/common/images/design_conf2.png)
+
+Both the grid and the configuration form are implemented using UI components.
+
+To change the grid view, you need to [customize the grid](#customize-the-grid) configuration by adding a custom `design_config_listing.xml` to your module.
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:40]:
```suggestion
To change the grid view, you must [customize the grid](#customize-the-grid) configuration by adding a custom `design_config_listing.xml` file to your module.
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 10]
```diff
@@ -0,0 +1,233 @@
...
+
+To change the available design settings you need to [customize the grid](#customize-the-grid) configuration by adding your custom `design_config_form.xml` in your module. If you add a new field, you must also declare it in `di.xml` on how it is processed and saved.
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:42]:
```suggestion
To change the available design settings, you must [customize the grid](#customize-the-grid) configuration by adding your custom `design_config_form.xml` file in your module. If you add a new field, you must also declare it in the `di.xml` file on how it is processed and saved.
```

Comment (jeff-matthews) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:42]: The last bit of this sentence is awkward

> ...you must also declare it in the `di.xml` file on how it is processed and saved.

Is the "... on how it is processed and saved." bit necessary or can you just remove it? If it's necessary, then please rewrite to clarify the relationship between the di.xml file and processing/saving.

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 11]
```diff
@@ -0,0 +1,233 @@
...
+
+## Customize the grid
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:44]:
```suggestion
## Customize the grid

```

Comment (jeff-matthews) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:44]: Need spaces before and after headings

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 12]
```diff
@@ -0,0 +1,233 @@
...
+The grid containing the configuration scopes is implemented using the [grid UI component]({{page.baseurl}}/ui_comp_guide/components/ui-listing-grid.html).
+
+To customize the grid view, take the following steps:
+
+1. In the `<your_module_dir>/view/adminhtml/ui_component` directory, add the empty `design_config_listing.xml`.
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:49]:
```suggestion
1. In the `<your_module_dir>/view/adminhtml/ui_component` directory, add the empty `design_config_listing.xml` file.
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 13]
```diff
@@ -0,0 +1,233 @@
...
+
+1. In the `design_config_listing.xml` file, create an element to which to add your customizations. For example, if you want to rename the column displaying the selected theme, your grid configuration must contain the following:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:54]: This entire code block must be indented 3 spaces to align properly under the preceding list number.

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 14]
```diff
@@ -0,0 +1,233 @@
...
+                <item name="config" xsi:type="array">
+                    <item name="label" xsi:type="string" translate="true">%New theme column name%</item>
+                </item>
+            </argument>
+        </column>
+    </columns>
+</listing>
+```
+
+Your `design_config_listing.xml` is merged with the same files from the other modules. So there is no need to copy their content, you only need to define changes. Even if you want to customize the existing entities, you only have to mention those options for which the values are customized.
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:69]:
```suggestion
Your `design_config_listing.xml` file is merged with the same files from the other modules, so there is no need to copy their contents. You only need to define changes. Even if you want to customize the existing entities, you only need to mention those options for which the values are customized.
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 15]
```diff
@@ -0,0 +1,233 @@
...
+
+For reference, view the grid configuration files of the Magento modules:
+
+-  `<Magento_Backend_module_dir>/view/adminhtml/ui_component/design_config_listing.xml`
+-  `<Magento_Theme_module_dir>/view/adminhtml/ui_component/design_config_listing.xml`
+
+If you add a certain field as an additional grid column, you also need to set the field’s `use_in_grid property` in the [field’s meta data](#add-fields-metadata) in `di.xml`.
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:76]:
```suggestion
If you add a certain field as an additional grid column, you must also set the field’s `use_in_grid` property in the [field’s meta data](#add-fields-metadata) in the `di.xml` file.
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 16]
```diff
@@ -0,0 +1,233 @@
...
+
+## Customize the design options
+
+These sections detail how to customize form configuration and field metadata.
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:80]: This sentence implies that some or all of the following headings are subordinate to this one, but the headings are at the same level (h2). 

Please confirm and update headings to h3 as needed:

```
## Customize the design options
### Customize the form configuration
### Add fields’ metadata
### Accessing the options values in backend models
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 17]
```diff
@@ -0,0 +1,233 @@
...
+
+The design configuration form is implemented using the form UI component.
+
+To customize the form view, take the following steps:
+
+1. Create an empty `design_config_form.xml` file in the `<your_module_dir>/view/adminhtml/ui_component/` directory.
+
+1. Add content similar to the following:
+
+```xml
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:92]: This entire code block must be indented three spaces to properly align under the preceding list number.

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 18]
```diff
@@ -0,0 +1,233 @@
...
+                </argument>
+            </field>
+        </fieldset>
+    </fieldset>
+</form>
+```
+
+Your custom fields and field sets will be available for all configuration scopes (website, store, and store view).
+
+Your `design_config_form.xml` is merged with the same files from the other modules. So there is no need to copy their content, you only need to add your customizations.
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:129]:
```suggestion
Your `design_config_form.xml` file is merged with the same files from the other modules, so there is no need to copy their contents. You only need to add your customizations.
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 19]
```diff
@@ -0,0 +1,233 @@
...
+
+To customize an existing entity, declare only those options, the values of which are customized, do not copy its entire configuration.
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:131]:
```suggestion
To customize an existing entity, declare only those options; the values of which are customized. Do not copy its entire configuration.
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 20]
```diff
@@ -0,0 +1,233 @@
...
+
+For example, if you only want to rename the **Other Settings** field set, your form configuration must contain the following:
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:133]:
```suggestion
For example, if you want to rename the **Other Settings** field set only, your form configuration must contain the following:
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 21]
```diff
@@ -0,0 +1,233 @@
...
+            <item name="config" xsi:type="array">
+                <item name="label" xsi:type="string" translate="true">Other Settings</item>
+
+            </item>
+        </argument>
+    </fieldset>
+</form>
+```
+
+To delete an existing field, or field set, in your `design_config_form.xml` use the following syntax:
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:151]:
```suggestion
To delete an existing field, or field set, in your `design_config_form.xml` file, use the following syntax:
```

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 22]
```diff
@@ -0,0 +1,233 @@
...
+
+For reference, view the form configuration files of these Magento modules:
+
+-  `<Magento_Backend_module_dir>/view/adminhtml/ui_component/design_config_form.xml`
+-  `<Magento_Catalog_module_dir>/view/adminhtml/ui_component/design_config_form.xml`
+-  `<Magento_Email_module_dir>/view/adminhtml/ui_component/design_config_form.xml`
+-  `<Magento_Swatches_module_dir>/view/adminhtml/ui_component/design_config_form.xml`
+-  `<Magento_Theme_module_dir>/view/adminhtml/ui_component/design_config_form.xml`
+
+## Add fields’ metadata
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:171]: Need spaces before and after headings

File: src/guides/v2.3/howdoi/admin/customize_admin.md [hunk 23]
```diff
@@ -0,0 +1,233 @@
...
+If in the design configuration form you add new fields, `<your_module_dir>/etc/di.xml`, you must specify their parent field sets and the path in the database. You can also declare the backend model used for processing the field values. If you do not specify any model, the default `Magento\Framework\App\Config\Value` model is used.
+
+The field declaration in a `di.xml` looks like the following:
```
Comment (jeff-matthews) [inline src/guides/v2.3/howdoi/admin/customize_admin.md:174]:
```suggestion
The field declaration in a `di.xml` file looks like the following:
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
Nice work @BarnyShergold! 

See editorial suggestions.

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 4 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:3]: I'd have preferred an all capital title but I'll go with it :)

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 5 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:14]: Ah I keep missing that one

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 19 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:129]: The first scans better for me. But you the boss. 😄

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 20 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:131]: I have to disagree with this one. "Only" and "want" - verb with adverb - should go together and not be split this way.

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 10 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:40]: I'm only transferring the article - but yet that last bit feels clumsy so I shortened it.

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 11 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:42]: Yeah - missed that one :)

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 12 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:47]: Resolved

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 13 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:52]: Resolved

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 14 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:67]: Resolved

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 15 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:74]: Resolved

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 20 above)
Comment (jeff-matthews) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:131]: Eh, semantics. It's not a big deal and I'm willing to overlook it.

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 16 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:78]: Agreed & resolved

File: src/guides/v2.3/howdoi/admin/customize_admin.md (hunk 17 above)
Comment (BarnyShergold) [reply src/guides/v2.3/howdoi/admin/customize_admin.md:90]: Resolved

**Comment by BarnyShergold** [comment]
@jeff-matthews - Accepted all changes accept 1 - please can you review & revert?

**Review by jeff-matthews — APPROVED** [review]
Well done @BarnyShergold!

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**